extensions/src/main/java/org/mapstruct/extensions/spring/converter/ConverterMapperProcessor.java

```diff
@@ -214,7 +222,7 @@ private boolean getLazyAnnotatedConversionServiceBean(
     final Types typeUtils = processingEnv.getTypeUtils();
     return typeUtils.directSupertypes(mapper.asType()).stream()
         .filter(
-            supertype -> typeUtils.erasure(supertype).toString().equals(SPRING_CONVERTER_FULL_NAME))
+            supertype -> typeUtils.erasure(supertype).toString().equals(BASE_AUTOMAPPER_FULL_NAME))
```

Comment:
I suspect this is the cause of the issue. Without this filter, we won't find the mappers defined in standard MapStruct style.